bin/smart-proxy-win-service

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(*Dir[File.expand_path("../../lib", __FILE__), File.expand_path("../../modules", __FILE__)])
+
+raise "win-service executable can only be used as a service in windows environment" unless RUBY_PLATFORM =~ /mingw/
+
+require 'smart_proxy'
+require 'win32/daemon'
+include Win32
+
+begin
+  # Logfile must be absolute on windows
+  logfile = "../../tmp/proxy.log"
```

Comment:
@witlessbird seems wrong to have a comment about needed absolute path on windows and then setting a relative path.